DATASETS.md

```diff
@@ -28,18 +28,20 @@ you can use the `sdgym.load_dataset` function passing the name of the dataset to
 In this example, we will load the `adult` dataset:
 
 ```python3
-In [1]: from sdgym.datasets import load_dataset
+In[1]:
```

Comment:
Lets remove this `In[x]` or `Out[x]`, those come from an `iPython` shell and it would be nice to clean them up.